dev-python/sympy/sympy-1.11.1-r2.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
This looks like a case for revbump via `git mv`, since you're fixing an incorrect dependency, correct?

Reply:
It looks like I didn't document myself on a a lot of background context, and I'm sorry for this. Let me remove the older revision...